### ISSUE TITLE
Add Tooltip to Close Button on Panel Sample.

### ISSUE THREAD

**Issue by sophiaFengMS** [microsoft/fluentui#14999]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

Open the Panel, hover on the close button, the tooltip 'Close' shows. Then try to access the tooltip, it disappears.

### Please provide a reproduction of the issue in a codepen:

Can be repro at the website: https://developer.microsoft.com/en-us/fluentui#/controls/web/panel
Open any panel in the Usage
#### Actual behavior:

The tooltip that appeared on the close button is non-hoverable.

#### Expected behavior:

The tooltip should be hoverable with the mouse, so that the user can stop them by hovering to them.

### Documentation describing expected behavior

As per MAS1.4.13, tooltip itself should be hoverable using mouse. Please find the below Example 2 : Hoverable tooltip
![image](https://user-images.githubusercontent.com/71104988/92947610-11e20a00-f40d-11ea-8077-fa457d9ba5ab.png)

**Comment by paulgildea** [comment]
@sophiaFengMS Thanks for the feedback.

I am able to hover over the `Tooltip` component provided on the documentation site: https://developer.microsoft.com/en-us/fluentui#/controls/web/tooltip
![image](https://user-images.githubusercontent.com/33229964/92952772-638e9280-f415-11ea-8d92-78d873dd65ff.png)
(sorry I wasn't able to capture the mouse cursor with my snipping tool)

Looking at the `Panel` sample it doesn't look like it's using the `Tooltip` component and is using the native browser tooltip. 
![image](https://user-images.githubusercontent.com/33229964/92952962-b49e8680-f415-11ea-8fdf-a4bd4586412c.png)

I'll use this issue to track making an update to the sample for `Panel` as I believe it is necessary for a developer to use the `Button` component and the `Tooltip` component together to achieve the desired experience.
